game/hiscore.cc

```diff
@@ -9,24 +9,32 @@
 #include <stdexcept>
 
 bool Hiscore::reachedHiscore(unsigned score, SongId songid, unsigned short level, std::string const& track) const {
-	if (score > 10000) throw std::logic_error("Invalid score value");
-	if (score < 2000) return false; // come on, did you even try to sing?
+	if (score > 10000)
+		throw std::logic_error("Invalid score value");
+	if (score < 2000)
```

Comment:
magic number, please make a constant for this. For now it looks alright, but once someone starts to refactor it might not be alright anymore

Reply:
I added constants for both numbers.

Reply:
@Baklap4 please take a look to the constant names.